library/src/features_from_query_parameter.rs

```diff
@@ -15,7 +17,9 @@ use indexmap::{IndexMap, IndexSet};
 
     for feature in features {
         // Eliminate XSS vuln
-        let safe_feature = feature.replace("*/", "");
+
+        let re = Regex::new(r"[*/]").unwrap();
```

Comment:
What's the reasoning here? This would surely replace any individual instances of `*` or `/` with an empty string?  I think that's ok actually, because none of the feature tokens contain those characters, but it is not doing the same thing as the original code.